cache/manager.go

```diff
@@ -136,6 +136,7 @@ func (cm *cacheManager) GetByBlob(ctx context.Context, desc ocispecs.Descriptor,
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
nit: this is a stylistic change, which adds to noise when looking at git history since nothing else in this file changed.

Reply:
Just a note that these extra lines were probably added by me while editing code & later reversing course. I'll clean them up.

Reply:
I tend to have stuff like this sneak in too. Thanks for cleaning it up and keeping the diff concise.